lib/spotlight/dor/indexer.rb

```diff
@@ -48,47 +47,52 @@ def solr_document resource
     def resource druid
       Harvestdor::Indexer::Resource.new harvestdor, druid
     end
-    
+
     private
     # def add_something_to_solr_doc sdb, solr_doc
-    #   solr_doc['some_field_name_tsi'] = 'some value'
+    #   solr_doc['some_field_name_tsi'] = 'some value'   # OR
+    #   insert_field solr_doc,'some_multivalued_field_name_ssim',array_of_values
     # end
 
+    def add_donor_tags sdb,solr_doc
+      donor_tags = sdb.smods_rec.note.select {|n| n.displayLabel == 'Donor tags'}.map {|n| n.content}
```

Comment:
Line is too long. [101/80]<br>Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Pass `&:content` as an argument to `map` instead of a block.